doc/lsst.pipe.tasks/deblending-flags-overview.rst

```diff
@@ -0,0 +1,153 @@
+
+Because the templates generated by the deblender are used to weight the flux from the image, the total flux in the footprints is conserved. This means that for isolated sources, the template that would be created by `SourceDeblendTask` is irrelevant, as all of the flux in the footprint would be returned. The result is an output catalog (in each band) with all of the parent blends at the top followed by all of the children deblended from one of the parents. So when this was the only deblender, selecting a set of unique sources was easy, you just cut on `deblend_nChild == 0`. This selected all of the isolated sources (from the parent section) and all of the deblended child sources.
+
+Multi-band deblender (`scarlet`)
+================================
+
+:py:class:`lsst.meas.extensions.scarlet.ScarletDeblendTask` is different, as it uses `scarlet` to create a *model* for each source in a blend. This is a philosophically different object, as there is no longer an assumption that all of the flux in the input image will be modeled by one of the children in the blend (this may change in the near future, but this is the current implementation). The results of the scarlet deblender will be biased by the assumptions that went into making the models, so it was decided that it would be a good idea to (by default) also model all of the isolated sources. This will allow comparisons of scarlet models of isolated sources to the un-modeled isolated source measurements to investigate the biases that scarlet is introducing and also gives users the option to choose between the un-modeled (parent) isolated source records and the scarlet model version of each isolated source. However this flexibility forced a change in the way that we select unique objects in a source catalog.
+
+Flags set by the deblender
+==========================
+
+Before we get into the flags set by pipe_tasks it is useful to understand the flags that are set in `SourceDeblendTask` and `ScarletDeblendTask` that relate to source selection.
```

Comment:
> Before we get into the flags set by `pipe_tasks`

A bunch of things:

`pipe_tasks` is an EUPS package, and that concept isn't relevant here. Usually, we replace `pipe_tasks` with `lsst.pipe.tasks` in end-user documentation because that's what users actually care about. But I'm also a big confused because this is a super-generic comment. Perhaps re-write this to suggest what category of tasks you're referring to?

Reply:
Also, a general comment that you should [**never** use "we" in end-user documentation](https://developers.google.com/style/person). Always write in the second person ("you..."). I'd suggest doing a search for "we" and re-writing those sentences.